packages/roosterjs-editor-plugins/lib/plugins/Paste/excelConverter/convertPastedContentFromExcel.ts

```diff
@@ -19,6 +19,15 @@ export default function convertPastedContentFromExcel(
     const { fragment, sanitizingOption, htmlBefore, clipboardData } = event;
     const html = excelHandler(clipboardData.html, htmlBefore);
 
+    // For Excel Online
+    if (fragment.childNodes.length > 0 && getTagOfNode(fragment.firstChild) == 'DIV') {
+        fragment.firstChild.childNodes.forEach(child => {
+            if (getTagOfNode(child) == 'TABLE') {
```

Comment:
Is it possible there are multiple TABLE child under DIV?

Reply:
Does it mean fragment.firstChild should always only have one child?

Reply:
In my copies, the fragment always was:

    <div>
        <head>
            <meta/>
            <meta/>
            <meta/>
        <head/>
        <body>
            <table>
                 [Table]
            </table>
        <body/>
    </div>

Reply:
I tried with your code ,and I see there are some META and TABLE under DIV, but no HEAD and BODY.

Anyway, I think we'd better check if the only child nodes are 0..n META and exactly 1 TABLE, only in that case we can replace the content.  In all other case just skip processing them, so that will be safer. Something like:

```ts
let hasTable = false;
firstChild.childNodes.every(x => {
  if (x.tagName == 'META') {
    return true;
  } else if (x.tagName == 'TABLE' && !hasTable) {
    hasTable = true;
    return true;
  } else {
    return false;
 });
```